src/main/java/com/fasterxml/jackson/core/util/Separators.java

```diff
@@ -15,9 +15,39 @@ public class Separators implements Serializable
 {
     private static final long serialVersionUID = 1;
 
+    public enum Spacing {
```

Comment:
Needs Javadoc, including `@since 2.16`

Reply:
Done; I've also added it to all the other new methods plus I added @deprecated to the ones which are replaced with the new Separators API.

Reply:
Excellent!